File: C#/백준/Bronze/9610. 사분면/사분면.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int[] q = new int[5];
            for (int i=0;i<a;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                if(aa[0]==0 || aa[1]==0){
                    q[0]++;
                }
                else if(aa[0]>0 && aa[1]>0){
                    q[1]++;
                }
                else if(aa[0]<0 && aa[1]>0){
                    q[2]++;
                }
                else if(aa[0]<0 && aa[1]<0){
                    q[3]++;
                }
                else{
                    q[4]++;
                }
            }
           for (int i=0;i<4;i++){
               Console.WriteLine($"Q{i+1}: {q[i+1]}");
           }
            Console.WriteLine($"AXIS: {q[0]}");
        }
    }
}

File: Python/백준/Bronze/9610. 사분면/사분면.py
cnt = [0,0,0,0,0] #축+4분면에 있는 좌표 개수
a = int(input())
for i in range(a):
    aa = list(map(int,input().split()))
    if aa[0]==0 or aa[1]==0:
        cnt[0]+=1
    elif aa[0]>0 and aa[1]>0:
        cnt[1]+=1
    elif aa[0]<0 and aa[1]>0:
        cnt[2]+=1
    elif aa[0]<0 and aa[1]<0:
        cnt[3]+=1
    else:
        cnt[4]+=1
for i in range(4):
    print(f"Q{i+1}: {cnt[i+1]}")
print(f"AXIS: {cnt[0]}")